encoder/irc_datatypes.h

```diff
@@ -20,7 +20,7 @@
 
 #ifndef _RC_DATATYPES_H_
 #define _RC_DATATYPES_H_
-
+#include <stdint.h>
```

Comment:
This file currently doesn't include any headers.
please move this to the *.c file where needed to continue with the current style.